internal/copy_handler.go

```diff
@@ -22,31 +23,76 @@ func HandleCopy(fromConfigFile string, toConfigFile string, backupName string, w
 	if fromError != nil || toError != nil {
 		return
 	}
-	var infos, err = getObjectsToCopy(backupName, from, to, withoutHistory)
+	infos, err := getObjectsToCopy(backupName, from, to, withoutHistory)
+	tracelog.ErrorLogger.FatalOnError(err)
+	isSuccess, err := StartCopy(infos)
+	tracelog.ErrorLogger.FatalOnError(err)
+	if isSuccess {
+		tracelog.InfoLogger.Println("Success.")
+	}
+}
+
+func StartCopy(infos []CopyingInfo) (bool, error) {
+	var maxParallelJobsCount = 8 // TODO place for improvement
```

Comment:
I' m almost sure that Golang has something like std::hardware_concurrency() in C++

Reply:
The problem is too many opened files, so std::hardware_concurrency() will not help

Reply:
I don't know which number i should choose and i didn't find the answer in google :)

Reply:
@Denchick 

>  i'd found out that 8 is optimal on my local machine.

WAL-G has good settings to control parallelism -- I wonder, why those aren't used here? 
Hard-coded `8` may be too big or too small depending on concrete case.